demo/src/app/components/tooltip/demos/basic/tooltip-basic.html

```diff
@@ -0,0 +1,12 @@
+<button type="button" class="btn btn-secondary" placement="top" ngbTooltip="Tooltip on top">
```

Comment:
I don't think we need `role="button"` on native `<button>` elements, ex.: http://stackoverflow.com/questions/26934719/do-i-need-role-button-on-a-button